frontend/pages/club/[club]/edit.js

```diff
@@ -312,13 +312,28 @@ class ClubForm extends Component {
   }
 
   render() {
-    const { authenticated, schools, majors, years, tags } = this.props
+    const { authenticated, userInfo, schools, majors, years, tags } = this.props
     const { club, invites, editMember } = this.state
 
     if (authenticated === false) {
       return <AuthPrompt />
     }
 
+    if (
+      authenticated &&
+      club &&
+      !userInfo.is_superuser &&
+      !userInfo.membership_set.some(m => m.code === club.code && m.role <= 10)
+    ) {
+      return (
+        <AuthPrompt title="Oh no!" hasLogin={false}>
+          You do not have permission to edit the page for {club.name}. To get
+          access, contact{' '}
```

Comment:
Maybe try `&nbsp;` instead of {' '}

Reply:
This is actually automatically formatted by [Prettier](https://prettier.io/), so I assume that this is the standard.